aiida/cmdline/commands/cmd_code.py

```diff
@@ -222,13 +222,9 @@ def show(code):
     from aiida.cmdline import is_verbose
 
     table = []
-    table.append(['PK', code.pk])
-    table.append(['UUID', code.uuid])
-    table.append(['Label', code.label])
-    table.append(['Description', code.description])
-    table.append(['Default plugin', code.default_calc_job_plugin])
-    table.append(['Prepend text', code.prepend_text])
-    table.append(['Append text', code.append_text])
+
+    for key in code.get_cli_options().keys():
+        table.append([key.capitalize().replace('_', ' '), getattr(code, key)])
```

Comment:
We should probably include the pk and UUID as it was before. This is also the reason the test is failing as the pk is not part of the `get_cli_options` and so wasn't shown
```suggestion
    for key in ('pk', 'uuid') + tuple(code.get_cli_options().keys()):
        table.append([key.capitalize().replace('_', ' '), getattr(code, key)])
```

Reply:
If you really want the PK, it's given by `verdi code list`

Reply:
Current commit adds PK and UUID. Not as nice a one liner, but formats them better in the table in all caps.